<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>667</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>969</width>
    <height>837</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>759</width>
    <height>610</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #22222e</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">backgound-color: rgb(0, 0, 0)</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>1101</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>759</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>759</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Beenance</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>251</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="usd_btn">
       <property name="font">
        <font>
         <family>Modern No. 20</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border: 2px solid;
	border-style:outset;
	border-radius:10px;
	border-color:rgb(0, 255, 255)
}

QPushButton:hover{
    background-color:rgb(128, 128, 128);
}</string>
       </property>
       <property name="text">
        <string>USD</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="euro_btn">
       <property name="font">
        <font>
         <family>Modern No. 20</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border: 2px solid;
	border-style:outset;
	border-radius:10px;
	border-color:rgb(0, 255, 255)
}

QPushButton:hover{
    background-color:rgb(128, 128, 128);
}</string>
       </property>
       <property name="text">
        <string>EURO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pound_btn">
       <property name="font">
        <font>
         <family>Modern No. 20</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border: 2px solid;
	border-style:outset;
	border-radius:10px;
	border-color:rgb(0, 255, 255)
}

QPushButton:hover{
    background-color:rgb(128, 128, 128);
}</string>
       </property>
       <property name="text">
        <string>Pound UK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tenge_btn">
       <property name="font">
        <font>
         <family>Modern No. 20</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border: 2px solid;
	border-style:outset;
	border-radius:10px;
	border-color:rgb(0, 255, 255)
}

QPushButton:hover{
    background-color:rgb(128, 128, 128);
}</string>
       </property>
       <property name="text">
        <string>Kz tenge</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>231</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="usd_line">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
border: 2px solid;
border-style:outset;
border-radius:10px;
border-color:white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="euro_line">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
border: 2px solid;
border-style:outset;
border-radius:10px;
border-color:white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pound_line">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
border: 2px solid;
border-style:outset;
border-radius:10px;
border-color:white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tenge_line">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
border: 2px solid;
border-style:outset;
border-radius:10px;
border-color:white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="Today">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border: 2px solid;
border-style:outset;
border-radius:10px;
border-color:white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
